### PR TITLE
Move TestLog.log to the TEMP directory

### LINKED ISSUE #1275: MSI uninstall leaves a remnant behind
### Describe the bug

When uninstalling (via the MSI), "\Program Files\ebpf-for-windows\TestLog.log" is left behind.

### OS information

_No response_

### Steps taken to reproduce bug

* Install from the MSI (see PR #1274 for instructions)
* Uninstall ebpf for Windows
* Look to see if "\Program Files\ebpf-for-windows" directory still exists

### Expected behavior

Directory should be removed

### Actual outcome

Directory is not removed since a log file remains in it

### Additional details

_No response_

### PR DESCRIPTION
**Pull request by dthaler**
Signed-off-by: Dave Thaler <dthaler@microsoft.com>

## Description

Make sure the ebpf-for-windows directory is cleaned up on uninstall.
Fixes #1275

## Testing

Manual (see #1279)

## Documentation

No impact

### CODE REVIEW

**Review by saxena-anurag — DISMISSED** [review]

File: scripts/common.psm1 [hunk 1]
```diff
@@ -19,7 +19,7 @@ function Write-Log
         if (($null -ne $TraceMessage) -and ![System.String]::IsNullOrEmpty($TraceMessage)) {
             $timestamp = (Get-Date).ToString('HH:mm:ss')
             Write-Host "[$timestamp] :: $TraceMessage"-ForegroundColor $ForegroundColor
-            Write-Output "[$timestamp] :: $TraceMessage" | Out-File "$PSScriptRoot\$LogFileName" -Append
+            Write-Output "[$timestamp] :: $TraceMessage" | Out-File "$env:TEMP\$LogFileName" -Append
```
Comment (Alan-Jowett) [inline scripts/common.psm1:22]: @shankarseal does this break existing code that gathers logs?

Comment (shankarseal) [reply scripts/common.psm1:22]: Yes I think line 254 of `config_test_vm.psm1` would need to be changed.

Comment (dthaler) [reply scripts/common.psm1:22]: Updated

Comment (dthaler) [reply scripts/common.psm1:22]: @shankarseal verified that testlog.log is in the artifacts now, please re-review

**Review by Alan-Jowett — APPROVED** [review]

**Review by saxena-anurag — APPROVED** [review]

**Pull request merged**